speech/api/README.md

```diff
@@ -37,10 +37,36 @@ See the
 [Cloud Platform Auth Guide](https://cloud.google.com/docs/authentication#developer_workflow)
 for more information.
 
+### Install the dependencies
+
+* If you're running the `speechrest.py` sample:
+
+    ```sh
+    $ pip install speechrest-requirements.txt
+    ```
+
+* If you're running the `speech_streaming.py` sample:
+
+    ```sh
+    $ pip install speech_streaming-requirements.txt
+    ```
+
 ## Run the example
 
-```sh
-$ python speechrest.py resources/audio.raw
-```
+* To run the `speechrest.py` sample:
+
+    ```sh
+    $ python speechrest.py resources/audio.raw
```

Comment:
Yeah - I was considering adding an underscore to speechrest, but got lazy 'cuz then I'd have to change the references to it -_-

Will update in a later PR